internal-api/src/main/java/datadog/trace/bootstrap/config/provider/stableconfigyaml/ConfigurationMap.java

```diff
@@ -0,0 +1,23 @@
+package datadog.trace.bootstrap.config.provider.stableconfigyaml;
+
+import java.util.HashMap;
+import java.util.Map;
+
+// ConfigurationMap represents configuration key-values found in stable configuration files
+public class ConfigurationMap extends HashMap<String, ConfigurationValue> {
+  public ConfigurationMap() {
+    return;
```

Comment:
```suggestion
    super();
```
this is more consistent with the other constructor